### DISCUSSION TITLE
Kendine göndermeli cümle sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#87] [Zeka Sorusu] [answered]
Bu cümlede ........ kelime, ......... hece ve .......... harf bulunmaktadır.

(Boşlukları yazı ile doldurunuz ve cümleye dahil olsunlar)

**Comment by FatihOnder81** [comment, marked as answer]
Bu cümlede on üç kelime yirmi beş hece ve elli dokuz harf bulunmaktadır hocam çok uğraşyım inşalalh doğrudur ya

**Reply by sonsuzus** [reply]
Harf sayısı tutmuyor gibi.

**Reply by FatihOnder81** [reply]
hocam tutuyor ben tek tek saydım bide kontrol amaçlı başkasınada saydırdım

**Reply by sonsuzus** [reply]
Noktayı ve boşlukları da harf olarak bakmıştım ama bu şekilde seninki de oluyor. Kabul ediyorum. 👍

**Reply by FatihOnder81** [reply]
hocam nokta boşluk falan hepsine karakter deniliyor sanırım ben o yüzden sadece harfleri saydım
